pkg/docker/config/config.go

```diff
@@ -17,8 +17,16 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// DockerAuthConfig exposes the auth information used by docker
+type DockerAuthConfig struct {
```

Comment:
This should document what `IdentityToken` is, and whether it can be set at the same time as `Username`/`Password`.

Reply:
Will do.